### ISSUE TITLE
Cannot find module './sketch/Sketch'

### ISSUE THREAD

**Issue by tobias-kuendig** [casesandberg/react-color#23]
When building my component using browserify it fails to resolve the `sketch/Sketch` module. 

The problem seems to be this lowercase path:

```js
var Sketch = require('./sketch/Sketch');
```
https://github.com/casesandberg/react-color/blob/master/src/components/Color.jsx#L10

in `src/components` there are two sketch folders. One lowercase and the other one is uppercase. In `lib/components` there is only the uppercase folder.

Changing the require statement from above to uppercase...

```js
var Sketch = require('./Sketch/Sketch');
```
... fixes the problem.

**Comment by SeeThruHead** [comment]
is this not fixed? it's broken in 1.3.5 for me

**Comment by catkins** [comment]
:wave: us too as reported in https://github.com/casesandberg/react-color/issues/84
